src/graph/ApolloGraphClient.ts

```diff
@@ -60,7 +60,8 @@ export class ApolloGraphClient implements GraphClient {
                                       variables?: Q,
                                       options?: any,
                                       current?: string): Promise<T> {
-        return this.executeQuery<T, Q>(resolveAndReadFileSync(queryFile, current), variables, options);
+        return this.executeQuery<T, Q>(
+            resolveAndReadFileSync(queryFile, current, {}), variables, options);
```

Comment:
Perhaps parameters should be an optional argument so you don't have to add passing `{}` everywhere.

Reply:
Good point. Adding.